### DISCUSSION TITLE
♿️ Improve Accessibility

### DISCUSSION THREAD

**Discussion by AntPolkanov** [SSWConsulting/SSW.Website#998] [Feedback/Ideas]
Currently compliance with Accessibility standards is a necessity and trend. Companies invest significant amounts of money in this area and, potentially, we can get our piece of the pie.

But there are some serious issues on our website in terms of accessibility.
For example:

1. The main menu is not accessible by the screen reader (except the "SSW TV" menu item)
2. The content of the 4-slide carousel at the top of the main page is not accessible
3. Headers like "_Australia's Leading .NET and Azure Consultants_", "_Upcoming Events_" or "_Contact US_" are not accessible
4. There are issues with contrast

Our home page is scored only 70% while the lowest acceptable level is 85% according to [AccessibilityChecker](https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fwww.ssw.com.au%2F&flag=au)

I am not saying that we should aim for WCAG 2.1 AAA, but A or even AA is achievable, I believe.

**Comment by bradystroud** [comment]
@jaydenalchin do you have any input here?

**Comment by jaydenalchin** [comment]
> @jaydenalchin do you have any input here?

Thanks Brady, @AntPolkanov is absolutely right and gives excellent points above 👏  And I love the dialog this will hopefully open. 

These are 100% standards we should be not only be adhering to in current projects but also standards we should be baking into design more going forward.

WCAG 2.1 AA compliance is commonly treated as the benchmark for web accessibility. We can look at AAA adherence as nice-to-have but it's still the gold standard that we should keep in mind and hit as much as possible. 

Huge improvements would be made by focusing on these things: 

- Minimum AA colour contrast
- Correct heading hierarchies
- Simplifying label & title text 
- Aria attributes for screen readers
- Tab targeting & fixing keyboard traps
- Bypass blocks

(But also we should replace the whole megamenu)

Thanks!

**Reply by bradystroud** [reply]
Do you want some internal time to do a proper accessibility review and make a bunch of PBIs? 

You could also do some kind of internal education e.g. CTF + a video + some rules

**Reply by jaydenalchin** [reply]
I'm currently doing an accessibility review for my client but yes that would be excellent work to be done when I'm able. 

I think at a minimum the dot points above could each be broken into their own approachable chunks / threads / issues / PBIs pending a proper A11Y review.

**Comment by sethdaily** [comment]
+1 to all of @AntPolkanov's points

I don't see a big downside to increasing the accessibility. There is also a good argument that increasing accessibility generally improves UI at the same time.
